runtime/compiler/x/codegen/J9CodeGenerator.cpp

```diff
@@ -84,6 +84,13 @@ J9::X86::CodeGenerator::CodeGenerator() :
        !TR::Compiler->om.canGenerateArraylets())
       cg->setSupportsInlineStringCaseConversion();
 
+   if (cg->getX86ProcessorInfo().supportsSSE3() &&
```

Comment:
It should be `supportsSSSE3`, note that SSE3 and SSSE3 are different.

Reply:
5630475

Reply:
Wow, good catch @0dvictor. Why would Intel name it like that 😂 the "S" "E" and "3" all look like the same character from a far!

Reply:
I know, what a wonderful naming 😂